.travis.yml

```diff
@@ -91,5 +91,8 @@ script:
   - set +e
 
 after_success:
-  - cp /tmp/coverage.txt .
-  - bash <(curl -s https://codecov.io/bash)
+  - |
+      if [[ -f /tmp/coverage.txt ]]; then
```

Comment:
might be better to test `$WITH_COVERAGE`  (so we'll get an error if coverage creation fails for some reason)

Reply:
Not done: after_success doesn't fail the build[1], so I don't think it would work like you expect.

[1] Unless it times out. see https://docs.travis-ci.com/user/customizing-the-build/#Breaking-the-Build